File: Projects/blinkit/backend/src/utils/hashing.js
import bcrypt from "bcrypt";

export const hashPassword = async (password) => {
  const salt = await bcrypt.genSalt(10);
  return await bcrypt.hash(password, salt);
};

export const verifyPassword = async (userPassword, dbPassword) => {
  return await bcrypt.compare(userPassword, dbPassword);
};

File: Projects/auth-fast-mern/fast-backend/app/utils/hashing.py
from passlib.context import CryptContext

pwd_context = CryptContext(schemes=["bcrypt"], deprecated="auto")

def hash_password(password:str) -> str:
    return pwd_context.hash(password)

def verify_password(user_password:str, db_password:str) -> bool:
    return pwd_context.verify(user_password, db_password)